homeassistant/components/devolo_home_network/config_flow.py

```diff
@@ -15,6 +15,7 @@
 from homeassistant.const import CONF_IP_ADDRESS, CONF_NAME, CONF_PASSWORD
 from homeassistant.core import HomeAssistant
 from homeassistant.helpers.httpx_client import get_async_client
+from homeassistant.helpers.service_info.zeroconf import ZeroconfServiceInfo
```

Comment:
Note: zeroconf is still being imported, but it is not declared as a dependency in manifest.json

Is that on purpose? or does it need fixing?

Reply:
Does it need more than the zeroconf key? Then it would need fixing ...

Reply:
I think `zeroconf_instance = await zeroconf.async_get_instance(hass)` on line 34 implies a hard dependency, as it still gets triggered on a user flow (not just a zeroconf flow)

Reply:
It's been added now...